packages/vscode-extension/src/project/metro.ts

```diff
@@ -350,11 +351,21 @@ export class Metro implements Disposable {
     // be 1, reanimated worklet runtime would get 2, etc.)
     // The most recent runtimes are listed first, so we can pick the first one with title
     // that that starts with "React Native Bridge" (which is the main runtime)
-    for (const page of listJson) {
-      if (page.reactNative && page.title.startsWith("React Native Bridge")) {
-        return page.webSocketDebuggerUrl;
+    const newDebuggerPages = listJson.filter(
+      (page) => page.reactNative && page.title.startsWith("React Native Bridge")
+    );
+    if (newDebuggerPages.length > 0) {
+      // Expo go apps would report at least two pages, first one being the Expo Go host runtime
+      // If we detect Expo Go package (using description field), we want to pick the second page
+      // otherwise we pick the first one
+      const description = newDebuggerPages[0].description;
+      const isExpoGo = description === EXPO_GO_BUNDLE_ID || description === EXPO_GO_PACKAGE_NAME;
```

Comment:
Why can it be either of them?

Reply:
BUNDLE_IDE is iOS concept and PACKAGE_NAME is Android one. It can be either depending on platform we run on